romdisk/uart.asm

```diff
@@ -124,23 +128,66 @@ uartsnd_main:
         ld l, a
         ; HL contains the string to output on the UART
         ; Get its length and save it in BC
-        push hl
-        call strlen
-        push bc
+
+        ; Filename in HL
+        ld b, h
+        ld c, l
+        ld h, O_RDONLY
+        OPEN()
+        or a
+        jp m, uart_file_error
```

Comment:
Use `open_error` instead, it is defined outside this file

Reply:
#fixed